repository/src/main/java/org/apache/atlas/repository/migration/DataMigrationStatusService.java

```diff
@@ -54,9 +59,14 @@ public DataMigrationStatusService(AtlasGraph graph) {
 
     public void init(String fileToImport) {
         try {
+            if(!validateFilePath(fileToImport)){
+                throw new AtlasBaseException("File Path is invalid");
+            }
             this.status = new MigrationImportStatus(fileToImport, DigestUtils.md5Hex(new FileInputStream(fileToImport)));
         } catch (IOException e) {
             LOG.error("Not able to create Migration status", e);
+        } catch (AtlasBaseException e) {
```

Comment:
Why catching exception here? write throws on method so that complete request will fail